### DISCUSSION TITLE
关于数据集的疑问

### DISCUSSION THREAD

**Discussion by Stark-zheng** [hpcaitech/ColossalAI#3371] [Community | Q&A]
提供的数据集链接里直找到了指令数据，没有回答和第二步的排序数据，想请问下这部分是没有提供吗？

**Comment by binmakeswell** [comment]
https://github.com/hpcaitech/ColossalAI/tree/main/applications/Chat#how-to-use
@Stark-zheng 您跟着步骤来即可，具体可以查看源码，开源的是完整代码。
